api/src/controllers/auth.ts

```diff
@@ -232,6 +233,21 @@ router.get(
 			req.session.redirect = req.query.redirect as string;
 		}
 
+		const accountability = {
+			ip: req.ip,
+			userAgent: req.get('user-agent'),
+			role: null,
+		};
+
+		emitAsyncSafe('oauth.provider', {
+			event: 'oauth.provider',
+			action: 'provider',
+			schema: null,
+			payload: null,
+			accountability: accountability,
+			user: null,
+		});
```

Comment:
We should probably include the actual provider used from `req.params.provider`

Reply:
> We should probably include the actual provider used from `req.params.provider`

as a new value?

Reply:
I was think if we could add it as part of the payload